### ISSUE TITLE
Print prediction to text file

### ISSUE THREAD

**Issue by sebastianruffberlin** [Deci-AI/super-gradients#1818]
### 💡 Your Question

Hi, my name is Sebastian, I am relatively new to the computer vision topics. I need help with Super Gradient and creating an code to save the predictions to a text file. I am using IPython and have had success creating file with bounding boxes and confidence ratings with this code:

from super_gradients.training import models
   ...: import cv2
   ...: import torch

device = 'cuda' if torch.cuda.is_available() else "cpu"
model = models.get("yolo_nas_s", pretrained_weights="coco")

img_path = r"C:/Users/Sebastian/images/440x/"
...: model.predict(img_path, conf=0.25).save(output_folder="C:/Users/Sebastian/images/yolonas_s/")

Not I need help with a correct code snippet to save the predictions of a bunch of images to a text file (doesnt matter if it is one text file for all or one textfile per image)

Can anyone help me with the code?

Thanks
Sebastian

**Comment by BloodAxe** [comment]
```
import os
import numpy as np
from super_gradients.training import models


model = models.get("yolo_nas_s", pretrained_weights="coco")

image_folder = "super-gradients/tests/data/tinycoco/images/train2017"
image_files = [os.path.join(image_folder, img) for img in os.listdir(image_folder)]

predictions = model.predict(image_files)

for image_file, result in zip(image_files, predictions):
    print(image_file)
    class_names = np.array(result.class_names)

    print(result.prediction.bboxes_xyxy)
    print(result.prediction.confidence)
    print(class_names[result.prediction.labels])

```
